### ISSUE TITLE
Post order should include first post

### ISSUE THREAD

**Issue by sadaszewski** [NodeBB/NodeBB#10268]
It is confusing and not very useful to keep the first post on the top when otherwise sorting posts from last to first. Could this be changed, potentially as an option?

**Comment by julianlam** [comment]
The reason this is maintained is because NodeBB's organization of posts separates the OP from the replies. This is despite the fact that OP itself is still a post.

I do agree with you that it is odd that when you change the sorting to "newest to oldest", that OP is floated to the top. The reason this is odd now is because the design of the topics encourages a linear perception of posts, and so that perception is broken when the posts appear out of order.

**Comment by julianlam** [comment]
I would not be averse to changing this behaviour in v3.x, and marking it as a breaking change.. @barisusakli ?

**Comment by barisusakli** [comment]
I think we need a better UX to tell that sorting effects the replies and not the OP, for example how they do on SO
![image](https://user-images.githubusercontent.com/2583394/221308509-1729f57b-3693-4478-91f1-87bb8708afb0.png)

I know a few communities where they embed a live stream in the OP and sort the replies by newest first and having OP at the op is desired in that case.

**Comment by julianlam** [comment]
Yes, and I think this was the behaviour with older forums as well. That's why while I have opinions on it, I don't want to break expectations either.

**Comment by mySebbe** [comment]
Current master still keeps the OP fixed and applies the sort order to replies, which matches the maintainer decision in this thread and preserves livestream/instruction topics. That means the original behavioral request is not an unresolved bug. I suggest closing it and, if still wanted, opening a narrower UX issue for the sort control's copy/placement so it clearly says that it sorts replies only.
